src/Options/index.js

```diff
@@ -241,6 +242,8 @@ export interface ParseServerOptions {
   playgroundPath: ?string;
   /* Callback when server has started */
   serverStartComplete: ?(error: ?Error) => void;
+  /* Rest representation on Parse.Schema https://docs.parseplatform.org/rest/guide/#adding-a-schema */
+  schema: ?MigrationsOptions;
```

Comment:
Can you rename the option type to `SchemaOptions`, since there won't be only migration related options in that group?